pkg/context/context.go

```diff
@@ -0,0 +1,18 @@
+package context
```

Comment:
I don't like the naming of this, it gets confusing.

Reply:
what about userContext? if this file will gather over time all types of contexts useful to zot, maybe just zotContext ?

Reply:
I would say requestContext or reqContext.

Reply:
package renamed to requestcontext .